docusaurus/docs/what-is-included/analytics.md

```diff
@@ -0,0 +1,57 @@
+
+It may be the case that you need to collect some more complex data for some more complex tags. For that, you're probably going to want to use the [Data Layer](https://developers.google.com/tag-manager/devguide). The utility module available in `www/shared/utils/google-tag-manager` has a `dataLayer` function that you can use in your components. An example would be a blog post page, where developers would inject blog related data such as its ID, title, author and date, and then marketeers can pick this data to populate events in `GTM`:
+
+```js
+import { dataLayer } from './shared/utils/google-tag-manager';
+
+const Blog = ({ blog }) => {
+    useEffect(() => {
+        dataLayer({
+            id: blog.id,
+            title: blog.title,
+            author: blog.author,
+            date: blog.date
+        });
+    }, [blog]);
+
+    return (
+        <div>{ /* ... */ }</div>
+    );
+};
+```
+
+## Urchin Tracking Module
+
+`UTM's` are parameters put in external urls that will help understand where did the redirect to your website come from. These are especially helpful for determing the effectiveness of marketing campaigns. Usually, you would use the official [`Campaign URL Builder`](https://ga-dev-tools.appspot.com/campaign-url-builder/) . But then, your URL would look something like this: `https://www.example.com/?utm_source=youtube&utm_medium=video&utm_campaign=winter_sale&utm_term=shirt`. A little bit ugly of a URL, isn't it? Well, `GTM` lets you make prettier URLs that collect the exact same data. We strongly recommend taking the approach described in this very detailed [article](https://www.getelevar.com/how-to/hide-google-utm-parameters-from-url/). Using this, your url would instead look something like `https://www.example.com/#youtube` and allow you to have the same valuable data provided by the `UTM's`, if setup properly. Go check it out your GA dashboard, in Real Time > Traffic Sources. The medium and sources you setup in GTM should be there. If not, `(none)` and `(direct)` will appear for each, respectively, and you need to take another look at the article to figure out what you did wrong.
+
+## Prompting for consent
+
+For privacy purposes, and in compliance with [GDPR](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation) and other regulations, apps need to ask the user for permission to collect user data. The boilerplate comes with a `<CookieBanner />` component that allows exactly this. Built with [`@enzsft/react-cookie-consents`](https://github.com/enzsft/react-cookie-consents), it will allow `GTM` to be initialized if the user has clicked that he accepts the data collection. It also saves the cookie for 365 days.
```

Comment:
`@enzsft/react-cookie-consents` is not being referenced anywhere else, is it not?

Reply:
It’s a left over, will remove.

Reply:
Fixed!